src/components/manage/Widgets/StyleWidget.jsx

```diff
@@ -0,0 +1,17 @@
+import ObjectWidget from '@plone/volto/components/manage/Widgets/ObjectWidget';
+
+const StyleWidget = (props) => {
+  const { block, onChange, schema } = props;
+
+  return (
+    <div className="styles-widget">
```

Comment:
Is there any chance to avoid this custom widget? If the classname is absolutely needed, maybe we can get the ObjectWidget to output a classname with the name of the field?

Reply:
Done